charts/fleet/charts/gitjob/values.yaml

```diff
@@ -1,10 +1,10 @@
 gitjob:
   repository: rancher/gitjob
-  tag: v0.8.12
+  tag: v0.8.13
 
 tekton:
   repository: rancher/tekton-utils
-  tag: v0.1.46
+  tag: v0.1.47
```

Comment:
As discussed, for reference: while both of those tags point to the same commit, creating a new tag triggers a new build and release, using the latest base image with updated OS packages.